01-syntax/01-language-basics/fizzBuzz.js

```diff
@@ -1 +1,13 @@
-// Your code here
+var digit = 1;
```

Comment:
leave here

```js
var digit;
```
and move assignment to `for` loop, right now it looks pretty strange ....